trollmoves/dispatcher.py

```diff
@@ -153,127 +157,53 @@
 import logging
 import os
 import signal
+import socket
+from datetime import datetime
 from queue import Empty
 from threading import Thread
 from urllib.parse import urlsplit, urlunsplit, urlparse
-import socket
 
 import yaml
-import inotify.adapters
-from inotify.constants import IN_MODIFY, IN_CLOSE_WRITE, IN_CREATE, IN_MOVED_TO
 from posttroll.listener import ListenerContainer
-from posttroll.publisher import NoisyPublisher
 from posttroll.message import Message
+from posttroll.publisher import NoisyPublisher
```

Comment:
Not necessary here, but we really should be using `posttroll.publisher.create_publisher_from_dict_config` so that all publisher features were available.

Reply:
This is now fixed along we more refactoring. Would be great if you could re-review :)